idl/chromadb/proto/debug.proto

```diff
@@ -0,0 +1,14 @@
+syntax = "proto3";
+
+package chroma;
+
+import "google/protobuf/empty.proto";
+
+message GetInfoResponse {
+  string version = 1;
+}
+
+service Debug {
+  rpc GetInfo(google.protobuf.Empty) returns (GetInfoResponse) {}
+  rpc TriggerPanic(google.protobuf.Empty) returns (google.protobuf.Empty) {}
+}
```

Comment:
suggestions welcome for a cleaner way to do this, I just want a way to trigger a panic within a service from a test